src/main/webapp/view/lecture-view.js

```diff
@@ -40,24 +44,20 @@ export default class LectureView {
   async initialize() {
     this.setLectureName();
 
-    // TODO: Make these private once event controller is created.
-    LectureView.eventController = new EventController();
-    LectureView.video = new Video(this.#lecture, LectureView.eventController);
-    // TODO: Move TranscriptArea initialization outside of initialize()
-    // and replace string parameter with a controller object.
-    LectureView.transcript =
-        new TranscriptArea(this.#lecture, LectureView.eventController);
-    LectureView.discussion =
-        new DiscussionArea(this.#lecture, LectureView.eventController);
+    this.#eventController = new EventController();
+    this.#video = new Video(this.#lecture, this.#eventController);
+    // TODO: Move TranscriptArea initialization outside of initialize().
```

Comment:
Should `TranscriptArea` still be moved?

Reply:
I'm unsure what that TODO means, I think it was added from a different PR. Do you still think we need this?

Reply:
I had that there because I didn't know if `TranscriptArea` should be created here at the time.

Reply:
Removed